sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -68,19 +90,23 @@ object AlluxioUtils {
                 s"from ${RapidsConf.ALLUXIO_PATHS_REPLACE.key} which requires only 1 rule " +
                 s"for each file path")
           } else if (matchedSet.size == 1) {
+            // check if the alluxio path exists, if not, throw out an exception to stop the job
+            checkAlluxioMounted(sparkSession, replaceMap(matchedSet.head))
```

Comment:
Is this called for every single file? If yes, it may suffer a performance penalty?

Reply:
Changed to only check in root paths.